indicators/NO_GJEN/R/NO_GJEN_004.qmd

```diff
@@ -0,0 +1,1467 @@
+  popskogecosysttype <- popecosysttype %>%
+    left_join(skog_region_bioclim, by = c('vegClimZoneLab','region'))
+  
+    
+  } else {
+     # Add the threshold for poor ecological condition in GRUK
+      aapne_region_bioclim <- skog_region_bioclim
+      aapne_region_bioclim$skog <- 0.8
+    
+        popskogecosysttype <- popecosysttype %>%
+    left_join(aapne_region_bioclim, by = c('vegClimZoneLab','region'))
+        
+  }
+  
+  refecosysttype <- read_csv(paste0('../data/ref',ecosysttype,'.csv'))
+
+  
+  # Combine the reference heights, forest heights and population heights into one dataset
+  ecosysttypeHts <- popskogecosysttype %>%
+    as_tibble() %>% 
+    dplyr::select(-geometry) %>%
+    left_join(refecosysttype, by=c('vegClimZoneLab', 'region')) %>%
+    gather(type, height, ref, pop, skog) %>%
+    group_by(`system:index`, region, vegClimZoneLab, type) %>%
+    summarise(height = mean(height)) %>%
+    pivot_wider(values_from = height, names_from = type) %>%
+    
+    # If population height is less than the reference value, then it is automatically "good" condition - ie. it inherits the reference height so that the rescaled value will be 1
+    mutate(pop = ifelse(pop < ref, ref, pop)) %>%
+    drop_na(pop, ref)
+  
+  ecosysttypeHts$region <- factor(ecosysttypeHts$region, levels=regionlvl)
+  ecosysttypeHts$vegClimZoneLab <- factor(ecosysttypeHts$vegClimZoneLab, levels=vegclimzonelvl)
+
+  assign(paste0(ecosysttype,"Hts"), ecosysttypeHts)
+  
+}
+
+```
+
+## 8. Spatial units
+
+<!--# Describe the spatial units that you rely on in your analyses. Highlight the spatial units (the resolution) that the indicator values should be interpretted at. Potential spatial delineation data should eb introduced under 7.1. Datasets. We recomend using the SEEA EA terminology opf Basic Spatial Units (BSU), Ecosystem Asses (EA) and Ecosystem Accounting Area (EAA). -->
+
+## 9. Analyses
+
+<!--# Use this header for documenting the analyses. Put code in seperate code chunks, and annotate the code in between using normal text (i.e. between the chunks, and try to avoid too many hashed out comments inside the code chunks). Add subheaders as needed. -->
+
+We will calculate the gjengroing index for each population polygon over Norway using a Sigmoid scaling function.
+
+```{r}
+
+# Define Sigmoid scaling function
+scaleSigmoid <- function(variable){
+  refLow <- min(variable)
+  refHigh <- max(variable)
+  thr <- (refHigh-refLow)/2
+  indicator_LowHigh <- (variable - refLow)/(refHigh - refLow)
+  indicator_LowHigh[indicator_LowHigh[] < 0] <- 0
+  indicator_LowHigh[indicator_LowHigh[] > 1] <- 1
+  indicator_sigmoid <- 100.68*(1-exp(-5*(indicator_LowHigh)^2.5))/100
+  return (round(indicator_sigmoid, 4))
+}
+
+```
+
+### 9.1 Scaled indicator values at polygon level
```

Comment:
I added two more figures in this section. Please have a look and see if they make sense.